src/Stratis.BitcoinD/Program.cs

```diff
@@ -27,8 +27,8 @@ public static void Main(string[] args)
 
                 var node = new FullNodeBuilder()
                     .UseNodeSettings(nodeSettings)
-                    .UsePowConsensus()
                     .UseBlockStore()
+                    .UsePowConsensus()
```

Comment:
why the order switch?

Reply:
BlockStore.cs: 114
// Throw if block store was initialized after the consensus.
// This is needed in order to rewind consensus in case it is initialized ahead of the block store.

Reply:
ah yes, cool